x-pack/plugins/transform/common/index.ts

```diff
@@ -38,3 +38,20 @@ export interface ResultData {
 export interface TransformEndpointResult {
   [key: string]: ResultData;
 }
+
+export interface DeleteTransformEndpoint {
```

Comment:
To be in line with `TransformEndpointRequest` suggest to rename this to `DeleteTransformEndpointRequest`.

Reply:
Fixed in https://github.com/elastic/kibana/pull/67922/commits/d3c37b5c0d03c2ca6e211a3ff2380154223c2f0d